source/NVDAObjects/UIA/winConsoleUIA.py

```diff
@@ -26,6 +27,21 @@ class consoleUIATextInfo(UIATextInfo):
 	#: to do much good either.
 	_expandCollapseBeforeReview = False
 
+	def __init__(self,obj,position,_rangeObj=None):
+		super(consoleUIATextInfo, self).__init__(obj, position, _rangeObj)
+		# Re-implement POSITION_FIRST and POSITION_LAST in terms of
```

Comment:
Why should POSITION_First and POSITION_LAST be limited to text which is on screen? Wasn't the intention to also expose text that is off screen, or is that outside the scope of this pr now?

Reply:
That is now out of scope.